.github/template-files/templates/pull_request_template_details.md

```diff
@@ -4,7 +4,7 @@
      please either tick them or use ~strikethrough~ so we know you've gone
      through the checklist. -->
 
-- [ ] Add a file to the `news` directory ([using the template](https://github.com/conda/conda-build/blob/main/news/TEMPLATE)) for the next release's release notes?
+- [ ] Add a file to the `news` directory ([using the template](https://github.com/conda/conda-build/blob/main/news/TEMPLATE.md)) for the next release's release notes?
```

Comment:
```suggestion
- [ ] Add a file to the `news` directory ([using the template](https://github.com/conda/conda-build/blob/main/news/TEMPLATE)) for the next release's release notes?
```

These templates come from `conda/infrastucture` so let's not touch them here. Thanks for noticing though!